.github/workflows/pull-request.yml

```
@@ -295,7 +295,117 @@
+            docker-compose -f docker-compose.dev.yaml logs redis-stack-server
+            docker-compose -f docker-compose.dev.yaml down -v
+            exit 1
+          fi
+
+          # Wait for TALAWA API to be healthy 
+          timeout=60
+          until docker-compose -f docker-compose.dev.yaml exec -T talawa-api-dev curl -sf "http://localhost:4000/health" 2>&1 || [ $timeout -eq 0 ]; do
+          echo "Waiting for API to start... ($timeout seconds remaining)"
+          sleep 1
+          ((timeout--))
+        done
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 372: could not find expected ':'

(syntax-check)